pubsub_nats_options.go

```diff
@@ -70,8 +68,15 @@ func natsOptClient(client natsClient) NATSOption {
 var ackMessage = []byte("ack")
 
 type natsSubscribeConfig struct {
-	queueGroup string
-	replier    func(msg *nats.Msg) []byte
+	queueGroup   string
+	replyTimeout time.Duration
+}
+
+func defaultSubscribeConfig() natsSubscribeConfig {
+	return natsSubscribeConfig{
+		queueGroup:   "",
+		replyTimeout: 5 * time.Second,
```

Comment:
I would put something greater like 60s

Reply:
I would be kind of surprised if clients would find it acceptable to potentially block for a minute waiting for a response. How about 20 seconds? :-P

Isn't it better to have a sane low default and let the client code specify an extreme timeout utilizing the `NATSOptSubscribeReplyTimeout` option? With that said, I think this option will go away once we allow a `Publisher` to encode a response deadline in their request. Right now this is basically a wild guess by the subscriber with respect to how long they think a client will be waiting for a response.

Reply:
Most of time people are willing to wait for a long time. They want the job done, they are not loading memes on facebook :) manipulate retries for 2m for instance. It offloads the burden of retrying in most cases.

Reply:
Donezo, bumped to 60s ⏫